tests/test_init_db.py

```
@@ -34,9 +34,9 @@
         allow_module_level=True,
     )
 
-from sqlalchemy.engine.url import make_url
+from sqlalchemy.engine.url import make_url  # noqa: E402
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)